src/config.js

```diff
@@ -1 +1,5 @@
-export const BASE_URL = '';
+export const BASE_URL = 'http://10.58.52.60:3000';
+
+export const API = {
+  kakao: 'users/auth/kakao/callback',
+};
```

Comment:
```suggestion
export const API = {
  kakao: `${BASE_URL}/users/auth/kakao/callback`,
};
```
이렇게 해주면, BASE_URL은 import 하지않아도 될것같네요.

Reply:
넵 말씀주신대로 변경하였습니다!